content/bash.rst

```diff
@@ -1,7 +1,7 @@
 Bash
 ####
 
-:menu_order: 155
+:menu_order: 015
```

Comment:
I tried it with just ``15`` but the menu ordering didn't turn out the way that it should be, so I reverted back